inst/components/card.scss

```
@@ -126,5 +131,16 @@
   position: fixed;
   inset: 0;
   background-color: rgba(var(--bs-body-color-rgb), 0.6);
+  backdrop-filter: blur(2px);
```

I _think_ this is needed for Safari

```suggestion
  backdrop-filter: blur(2px);
  -webkit-backdrop-filter: blur(2px);
```